app/src/main/java/com/hammer/talkbbokki/data/di/NetworkModule.kt

```diff
@@ -0,0 +1,46 @@
+package com.hammer.talkbbokki.data.di
+
+import com.google.gson.GsonBuilder
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between